### DISCUSSION TITLE
Using/configuring the LoCo positioning node without USB

### DISCUSSION THREAD

**Discussion by ghost** [bitcraze/discussions#1022] [Q&A] [answered]
My LoCo node positioning board broke. The USB connector lifted off, taking the pads with it, leaving no option to solder it back on. I was wondering, is it possible to configure the node's ID and operating mode through the debug port, using a programmer like the ST-Link or a simple USB-UART bridge? If all else fails, could I just change some constants in the anchor firmware to change the ID manually and flash it?

**Comment by ataffanel** [comment, marked as answer]
Hi,

If you enable the flag [```USE_FTDI_UART```](https://github.com/bitcraze/lps-node-firmware/blob/73f8960c728a3030b331b0abe19f73239bf6f38b/Makefile#L7) when compiling the node, it will used the FTDI-cable header (connector P3) on the node instead of the USB uart. This will allow you to get access to the configuration menu. The UART is setup with a baudrate of 115200 (115200-8-N-1).

**Reply by ghost** [reply]
Excellent, thank you! Another question, though: I don't have an FTDI cable, only a 4 pin serial adapter. Do I need the RTS and CTS pins on the FTDI port for the serial connection?

**Reply by ataffanel** [reply]
No you do not, flow control is not setup by default. Any 3.3V UART will work if connected to the right pins :-).

**Reply by ghost** [reply]
Ah, perfect! I managed to get it to work, and I can configure everything! Thank you so much for the help!
